settings.gradle.kts

```
@@ -12,6 +12,18 @@ subdir("entry-generation") {
     include("godot-annotation-processor")
 }
 
+subdir("entry-generation") {
+    include("godot-compiler-native-plugin")
+}
+
+subdir("entry-generation") {
+    include("godot-compiler-plugin")
+}
+
+subdir("entry-generation") {
+    include("godot-compiler-plugin-common")
+}
+
```

You can collapse these lines into a single `subdir`.

```suggestion
subdir("entry-generation") {
    include("godot-compiler-native-plugin")
    include("godot-compiler-plugin")
    include("godot-compiler-plugin-common")
}
```

---

:+1: